scripts/ie11-styles-post-processing.js

```diff
@@ -0,0 +1,52 @@
+const presetEnv = require('postcss-preset-env');
```

Comment:
No changes needed - wonder if there is a webpack plugin that does this instead of needing to write our own tooling.

Reply:
> Without any configuration options, PostCSS Preset Env enables Stage 2 features.

I think we've made at least some [basic changes to the config](https://github.com/patternfly/patternfly-react-seed/pull/21/files#diff-751169c7d48b495927919cedb63dbd63R17) (although this is still up in the air)

Will definitely keep that in mind though. I guess one benefit of the custom script is that an app that's not running webpack for the build could use it as a guide.

Reply:
Actually, looks like there is webpack support already, and you can specify options there, too. 

https://github.com/csstools/postcss-preset-env/blob/master/INSTALL.md#webpack

I'll work on getting this going with the current approach since it's already that way, and then we can look at converting afterward.